src/main/resources/presets/protocols/bd.yaml

```diff
@@ -1,190 +1,670 @@
-                        - type: inflate
-                          fraction: 0.15
-                      logX: true
-                      minimalSample: 20
-                      fallbackThreshold: 1.0
-                    - type: group_operator_cumtop
-                      share: 0.85
-                      round: Down
-          - type: filter_groups
-            groupingKeys:
-              - allTags:Cell
-            predicates:
-              - metrics:
-                  - type: group_metric_nunique
-                    reportHist:
-                      log: true
-                      binNumber: 0
-                      minBinWidth: 0.2
-                    keys:
-                      - allTags:Molecule
-                operator:
-                  type: group_operator_range
-                  lower: 3.0
-  assemble:
-    inheritFrom: assemble-with-consensus-base
     overrides:
-      clnaOutput: true
-      cloneAssemblerParameters:
-        postFilters:
-          - type: filter_in_groups
-            isolationKeys:
-              - geneLabel:ReliableChain
-              - allTags:Cell
-            nestedFilter:
-              type: filter_groups
-              groupingKeys:
-                - clone
-              predicates:
-                - metrics:
-                    - type: group_metric_nunique
-                      keys:
-                        - allTags:Molecule
-                  operator:
-                    type: group_operator_cumtop
-                    share: 0.95
-          - type: filter_in_groups
-            isolationKeys:
-              - allTags:Cell
-            nestedFilter:
-              type: filter_groups
-              groupingKeys:
-                - clone
-              predicates:
-                - metrics:
-                    - type: group_metric_nunique
-                      keys:
-                        - allTags:Molecule
-                  operator:
-                    type: group_operator_cumtop
-                    share: 0.99
-          - type: filter_groups
-            groupingKeys:
-              - allTags:Cell
-              - clone
-            predicates:
-              - metrics:
-                  - type: group_metric_nunique
-                    keys:
-                      - allTags:Molecule
-                operator:
-                  type: group_operator_range
-                  lower: 2.0
-          - type: filter_groups
-            groupingKeys:
-              - allTags:Cell
-            predicates:
-              - metrics:
-                  - type: group_metric_nunique
-                    keys:
-                      - clone
-                operator:
-                  type: group_operator_range
-                  upper: 4.0
-  exportClones:
-    inheritFrom: exportClones-single-cell-with-umi-base
+      tagTransformationSteps:
+        - type: mapTags
+          name: "CLS1"
```

Comment:
It is better to have more human-readable names for transformations, as they make their way into reports.